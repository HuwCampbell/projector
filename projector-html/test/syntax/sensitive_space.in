(({|<a><marquee/></a>|})   (_.basin))