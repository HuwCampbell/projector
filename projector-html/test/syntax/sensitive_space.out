Template
  ()
  Nothing
  (TEApp
     ()
     (TENode
        ()
        (THtml
           ()
           [ THtmlWS
               ()
               [ TElement
                   ()
                   (TTag () "a")
                   []
                   (THtml () [ TVoidElement () (TTag () "marquee") [] ])
               ]
           ]))
     (TEPrj () (TEHole ()) TId { unTId = "basin" }))